appserver/extras/embedded/shell/glassfish-embedded-static-shell/pom.xml

```diff
@@ -155,13 +155,6 @@
             <type>zip</type>
             <optional>true</optional>
         </dependency>
-        <dependency>
-            <groupId>org.glassfish.main.packager</groupId>
-            <artifactId>glassfish-cluster</artifactId>
```

Comment:
I actually don't understand why there was any class missing in the first place, because these dependencies are only used for bootstrapping embedded glassfish launcher, and then common classloader is constructed from the entire modules directory. However, `glassfish-cluster` was not a dependency of `embedded-all`, so I threw it away.